packages/react/src/file-upload/docs/overview.mdx

```diff
@@ -113,7 +113,23 @@ In this example, the file is instantly uploaded to a file hosting service, and t
 
 ## Multiple files
 
-Selecting multiple files is also supported with FileUpload. Simply add `multiple={true}`, and you can select as many files as you want. You can also set `maxFiles` to limit how many files can be selected.
+Selecting multiple files is also supported with FileUpload. Simply add `multiple={true}`, and you can select as many files as you want. You can also set `maxFiles` to limit how many files can be selected. Acceptable files that are marked as being over the upload limit will backfill accepted files if any are removed.
```

Comment:
> Acceptable files that are marked as being over the upload limit will backfill accepted files if any are removed.

I think I understand what this is trying to say, but having a hard time actually understanding it

Reply:
max 3 files

```
[ ] [ ] [ ]
```

You upload 5

```
[x] [x] [x] x x
```

2 are in an error state due to too many files but are still valid for use if space opens up

```
[Y] [Y] [Y] N N
```

you delete the 3rd file opening up some space

```
[Y] [Y] [ ] N N
```

the 4th file goes from invalid to valid since the 3rd space has opened up.

```
[Y] [Y] [Y] N
```

Reply:
Happy to receive suggestions on better ways to word it.

Reply:
I'll need to update this text now since this funcitonality doesn't exist anymore

- [ ] Update documentation text